extensions/sql-database-projects/src/models/project.ts

```diff
@@ -529,6 +538,15 @@ export class Project implements ISqlProject {
 		return version;
 	}
 
+	/**
+	 * Gets the default database collation set in the project.
+	 *
+	 * @returns Default collation for the database set in the project.
+	 */
+	public getDatabaseDefaultCollation(): string {
+		return this.evaluateProjectPropertyValue(constants.DefaultCollationProperty, constants.DefaultCollation);
```

Comment:
Here I'm not really sure what DacFX will do during deployment, if collation is not specified in the project - is it going to suggest `SQL_Latin1_General_CP1_CI_AS` or default to server-level (`master`) collation?

The UI in Visual Studio shows `SQL_Latin1_General_CP1_CI_AS` when project does not have the property and I decided that instead of having consumers to guess the default or hardcode General_CP1 everywhere - we will return it from here. This does imply that consumer will not be able to distinguish "property not set" vs. "property is set to SQL_Latin1_General_CP1_CI_AS", but same is true for VS users in the UI, so I assumed it's fair.